src/formatDate.js

```diff
@@ -8,7 +8,69 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // Set the new divider type
+  const newDivider = toFormat[toFormat.length - 1];
```

Comment:
Remove all comments, your variable describes what it does.

If you need to write comments to provide more information think about how to improve your solution so every developer can understand what your code does instead